pkgs/tools/networking/httplz/default.nix

```diff
@@ -19,7 +19,7 @@ rustPlatform.buildRustPackage rec {
 
   cargoBuildFlags = [ "--bin httplz" ];
   cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
If you can, it'd be nice to ask upstream to start checking in the cargo.lock with their releases, as this is recommended best practice from the Cargo Book:
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md#compiling-rust-applications-with-cargo

Then we could drop this ugly patch entirely.

Reply:
I know but at least of the devs is a heretic. :disappointed:  https://github.com/thecoshman/http/issues/84

> Moreover, it'd let the nixpkgs-update bot generate and send PRs to do this automatically.

I know :cry: 

I'm sure there's a lot of other nice packages with the same features. Maybe we should just drop this package and move on.